src/components/Sidebar/IconLink.jsx

```diff
@@ -2,36 +2,28 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { NavLink } from 'react-router-dom';
 import classNames from 'classnames';
-import { Icon } from '@edx/paragon';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 const IconLink = props => (
```

Comment:
nit: I have a slight preference for object destructuring, so instead of props, 
`({ isExpanded, icon, title, to })`

But, you didn't write this originally, so nbd.

Reply:
good point let me change this as well!